conda/recipes/ucx-py/meta.yaml

```diff
@@ -39,7 +39,7 @@ requirements:
 
 test:
   requires:
-    - cudatoolkit {{ cuda_version }}.*
+    - cuda-version ={{ cuda_version }}
```

Comment:
Thinking about this more, maybe we can drop the test requirements altogether?

Since all we are testing here is that `import ucp` works and that doesn't require CUDA transport support from `ucx`, maybe no additional requirements are needed for tests

Any tests needing the CUDA transport happen independently of the package build (so already install the needed dependencies)

Thoughts?

```suggestion
```

Reply:
Yes, removing those make sense. UCX-Py does not have CUDA as a hard-dependency, using it for host transfers only is totally valid. Done in https://github.com/rapidsai/ucx-py/pull/967/commits/dcac26dbf52f3888b9918a186f12af23fa18bf8e .